### ISSUE TITLE
Crash when harvesting grass with the aura field creator

### ISSUE THREAD

**Issue by DynamOdin** [thephoenixlodge/Heavens-of-Sorcery#89]
When harvesting grass with the aura field creator, Minecraft crashes. The log identifies the mod Dropt as the culprit.
[crash-2020-09-07_14.15.27-server.txt](https://github.com/thephoenixlodge/Heavens-of-Sorcery/files/5183061/crash-2020-09-07_14.15.27-server.txt)

**Comment by HalestormXV** [comment]
If this is indeed the case and is an occurrence and can, in-fact, be attributed to the Grass block, a new script will likely need to be added along with a dropt grassBlock.json entry made in the dropt folder so that any "grass" drops are nulled and replaced with an empty drop table. 

Mind you I haven't tested this nor have I meddled with Dropt in quite some time and have no idea if it would in fact work or if quests in the pack requires dropped grass (to be honest I can't remember although I am not incredibly far in it just yet either) 
I haven't made an Aura Field Creator yet nor have I even tested this. Perhaps I will give it a go later.

Assuming, Dropt can handle it and it is the actual drop from the "grass", you'd have to use the selective_removal from the DroptApi in your script.  https://dropt.readthedocs.io/en/latest/examples/selective_removal/ and create a JSON entry so that the loottable is set to []. 

If grass is used in the pack perhaps a Cauldron entry or something of that nature would need to be used so it can seamlessly be acquired.

TLDR: I haven't messed with Dropt in a long time and haven't been able to test this out myself or check the report/confirm the crash. Nor have I gotten to this point in the pack yet. If the grass is causing the issue then an entry and script may need to be written to "null" out the grass drop. Try at your own Risk as a temp fix until one of the pack creators sees this and can verify.
